src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -206,16 +208,18 @@ export class ExamParticipationComponent implements OnInit, OnDestroy, ComponentC
         }
     }
 
-    get activeExerciseIndex(): number {
-        if (!this.activeExercise) {
-            return 0;
+    get activePageIndex(): number {
+        if (!this.activeExamPage || this.activeExamPage.isOverviewPage) {
+            return -1;
         }
-        return this.studentExam.exercises!.findIndex((examExercise) => examExercise.id === this.activeExercise.id);
+        return this.studentExam.exercises!.findIndex((examExercise) => !this.activeExamPage.isOverviewPage && examExercise.id === this.activeExamPage.exercise!.id);
```

Comment:
Second this.activeExamPage.isOverviewPage seems unnecessary to me, is it possible to reach there if this.activeExamPage.isOverviewPage is true?

Reply:
done